README.md

```diff
@@ -23,14 +23,14 @@ defer span.Finish()
 ## Examples
 
 ```
-docker run -d -p 9411:9411 openzipkin/zipkin
+make run-jaeger
 go run examples/basic/main.go
 go run examples/httpServer/main.go
 go run examples/httpServer-middleware/main.go
 go run examples/httpGoKit-middleware/main.go
 ```
 
-To watch traces you just have to hit http://localhost:9411/
+To watch traces you just have to hit http://localhost:16686/search
 
 ## Features
```

Comment:
Info about required ENVs for `jaeger` would be useful

Reply:
they are part of the examples

Reply:
will add more description